# Reff: http://wiki.ros.org/costmap_2d
robot_base_frame: base_link
footprint: [[0.14, 0.14], [0.14, -0.14], [-0.14, -0.14], [-0.14, 0.14]]
rolling_window: true

# Configuration for our new custom layer
energy_zone_layer:
  enabled: true
  critical_battery_threshold: 30.0
  zone_cost_value: 200 # The cost to paint inside the zone (0-252)
  zones:
    - name: "carpet_main_hall"
      min_mx: 70
      max_mx: 100
      min_my: 70
      max_my: 100

inflation_radius: 0.1
cost_scaling_factor: 10.0

track_unknown_space: true
observation_sources: scan
scan: {sensor_frame: laser, data_type: LaserScan, topic: scan, marking: true, clearing: true}